Replace the removed default images for matrixview screensaver 
with those found in /usr/share/pixmaps/faces

diff -Nurp -x '*~' -x '*.orig' rss-glx_0.9.0/utils/rss-glx_install.pl rss-glx_0.9.0/utils/rss-glx_install.pl
--- rss-glx_0.9.1/utils/rss-glx_install.pl	2009-05-19 07:36:29.000000000 -0700
+++ rss-glx_0.9.1/utils/rss-glx_install.pl	2009-05-27 17:41:41.038691000 -0700
@@ -26,7 +26,7 @@ $screensavers->{'sundancer2'} 	= '  GL: 
 $screensavers->{'biof'} 	= '  GL:                   "BioF"  biof --root                                 \\n\\';
 $screensavers->{'busyspheres'} 	= '  GL:            "BusySpheres"  busyspheres --root                          \\n\\';
 $screensavers->{'spirographx'} 	= '  GL:            "SpirographX"  spirographx --root                          \\n\\';
-$screensavers->{'matrixview'} 	= '  GL:             "MatrixView"  matrixview --root                           \\n\\';
+$screensavers->{'matrixview'} 	= '  GL:             "MatrixView"  matrixview --root -i /usr/share/pixmaps/faces \\n\\';
 $screensavers->{'lorenz'}	= '  GL:                 "Lorenz"  lorenz --root                               \\n\\';
 $screensavers->{'drempels'}	= '  GL:               "Drempels"  drempels --root                             \\n\\';
 $screensavers->{'feedback'}	= '  GL:               "Feedback"  feedback --root                             \\n\\';
